jekyll/_cci2/certificates.adoc

```diff
@@ -193,8 +193,26 @@ Once you have a valid certificate and key file in `.pem` format, you must upload
 
 . Click "Save" at the bottom of the settings page and restart when prompted
 
-More information is available https://letsencrypt.readthedocs.io/en/latest/using.html#manual[here].
-
 Ensure the hostname is properly configured from the Management Console (`<circleci-hostname>.com:8800`) **and** that the hostname used matches the DNS records associated with the TLS certificates.
 
 Make sure the Auth Callback URL in GitHub/GHE matches the domain name pointing to the Services machine, including the protocol used, for example `**https**://info-tech.io/`.
+
+=== Adding the certificate to Replicated
+
+ Replicated Management Console(`<circleci-hostname>.com:8800`) runs on different web server, so you need to apply the above certificate for Replicated as well.
+
+. To do so, navigate to `https://hostname:8800/console/settings#tls-key-cert`
+
+. Upload your newly generated certificate and key
+
+. Click "Save" at the bottom of the page and restart the Replicated UI when prompted
+
+Even if you can't access the Replicated Page with the hostname, you still can access by IP address.
```

Comment:
@ganezasan just wondering, is this note in here because it often doesn't work? Just thinking we could make this a note so:

**Note:** Sometimes you will be unable to access the Replicated settings page listed above using the hostname. If this is the case you will be able to access using the IP address.

Reply:
@rosieyohannan Sorry, I forgot to add the condition of this. This means, when the TLS certification has expired, you can't access the Replicated UI with the hostname(domain name), but you still can access it by the IP address. So they can update the TLS certification on the UI.